### PR TITLE
UI NTC selector for adequacy patch

### PR DESCRIPTION
**Pull request by Milos-RTEi**

### CODE REVIEW

**Comment by Milos-RTEi** [comment]
Apparently in Antares you cannot use upper case letters in generaldata.ini file. 
Like: "section->add("set-11-**NTC**-to-null-for-adq-patch", setToZero11LinksForAdequacyPatch);"
I figured that out after 2 hours of debugging :(

**Comment by Milos-RTEi** [comment]
If anybody has prettier/simpler/more understandable name for the UI interface than ""Adequacy patch NTC 1->2"" &""Adequacy patch NTC 1<->1"" please make a suggestion!
I don't like mine either.

**Comment by rtei-pw** [comment]
> If anybody has prettier/simpler/more understandable name for the UI interface than ""Adequacy patch NTC 1->2"" &""Adequacy patch NTC 1<->1"" please make a suggestion! I don't like mine either.

"no ntc between physical areas inside and outside adq patch", 
"no ntc among physical areas outside adq patch"

File: src/libs/antares/study/parameters.h [hunk 1]
```diff
@@ -505,6 +505,13 @@ class Parameters final
     SimplexOptimization simplexOptimizationRange;
     //@}
 
+    //! Transmission capacities options for links between area type 1 and area type 2, for adequacy
```
Comment (rtei-pw) [inline src/libs/antares/study/parameters.h]: maybe detail a bit the doc.
like no ntc for link between physical ares inside and outside adq patch
no ntc for link between physical areas outside adq patch

**Review by rtei-pw — DISMISSED** [review]

File: src/libs/antares/study/parameters.h (hunk 1 above)
Comment (Milos-RTEi) [reply src/libs/antares/study/parameters.h]: Comments improved!

**Review by rtei-pw — APPROVED** [review]

**Review by hugo-antoine-rtei — CHANGES_REQUESTED** [review]
I think the only thing that's missing now is to plug this into the SetNTCForAdequacyFirstStep call in the src/solver/optimisation/opt_gestion_des_bornes_cas_lineaire.cpp file

**Review by hugo-antoine-rtei — APPROVED** [review]

**Pull request merged**